frontend/app/route-helpers/apply-adult-route-helpers.server.ts

```diff
@@ -33,6 +34,7 @@ export interface ApplyAdultState {
   readonly disabilityTaxCredit?: DisabilityTaxCreditState;
   readonly livingIndependently?: LivingIndependentlyState;
   readonly allChildrenUnder18?: AllChildrenUnder18State;
+  readonly childInformation?: ChildInformationState;
```

Comment:
It should reside in the `adultChildState` but this doesn't exists at the moment. Please work on this with @AdamP-HRSDC-RHDCC  and @shawn320.